lua/rulebook/data/add-ignore-rule-comment.lua

```diff
@@ -11,6 +11,13 @@
 
 ---@type table<string, ruleIgnoreConfig>
 M = {
+	["ast-grep"] = {
+		comment = "// ast-grep-ignore: %s",
```

Comment:
If I understand correctly, ast-grep works in many different languages. This means you cannot hardcode `//` as comment syntax here, as it would fit example result in invalid code when using ast-grep in Python or lua. 

See the entries for `biome` or `harper` further below how to dynamically determine comment syntax.